### PR TITLE
Add create on Panel model

### PR DESCRIPTION
**Pull request by arincon9**

### CODE REVIEW

File: lib/qualtrics_api/panel.rb [hunk 1]
```diff
@@ -15,17 +16,38 @@ def import_members(mbs)
...
-    def create(member)
+    def create_member(member)
       members.create(member)
     end
 
+    def create
+      payload  = create_attributes.transform_keys { |key| create_attributes_mappings[key] }
+      response = QualtricsAPI.connection(self).post("mailinglists", payload).body["result"]
+
+      return QualtricsAPI::Panel.new(self.attributes.merge(id: response["id"]))
```
Comment (kcotugno) [inline lib/qualtrics_api/panel.rb]: 👮 Unnecessary return.

**Review by kcotugno — APPROVED** [review]
LGTM! 👍

File: lib/qualtrics_api/panel.rb [hunk 2]
```diff
@@ -15,17 +16,38 @@ def import_members(mbs)
...
+    end
+
     private
 
+    def create_attributes
+      self.attributes.except(*(attributes_mappings.keys - create_attributes_mappings.keys))
```
Comment (lhwalsh) [inline lib/qualtrics_api/panel.rb]: We might be able to use slice here instead of except

File: spec/lib/panel_spec.rb [hunk 3]
```diff
@@ -60,18 +60,42 @@
     it 'is delegated to the panel member collection' do
       collection_double = double()
       allow(subject).to receive(:members) { collection_double }
```
Comment (lhwalsh) [inline spec/lib/panel_spec.rb:62]: should these two lines be put in a before do?

File: spec/lib/panel_spec.rb [hunk 4]
```diff
@@ -60,18 +60,42 @@
...
       subject.import_members([])
     end
   end
 
-  describe '#create' do
+  describe '#create_member' do
     let(:member) { QualtricsAPI::PanelMember.new }
     it 'is delegated to the panel member collection' do
       collection_double = double()
       allow(subject).to receive(:members) { collection_double }
```
Comment (lhwalsh) [inline spec/lib/panel_spec.rb:73]: should these two lines be put in a before do?

**Review by lhwalsh — APPROVED** [review]
LGTM aside from a few small changes

**Comment by arincon9** [comment]
@kcotugno @lhwalsh Comments have been addressed

**Pull request merged**